doc/blobfuse2.md

```diff
@@ -0,0 +1,29 @@
+## blobfuse2
```

Comment:
Do we really need to checkin all these md files ?

Reply:
Yea doesn't hurt to keep them. Customers might also just go through these if they didn't access the MS Docs